libraries/Microsoft.Bot.Schema/Teams/Generated/MessageActionsPayload.cs

```diff
@@ -60,7 +60,8 @@ public MessageActionsPayload()
         /// <param name="mentions">List of entities mentioned in the
         /// message.</param>
         /// <param name="reactions">Reactions for the message.</param>
-        public MessageActionsPayload(string id = default(string), string replyToId = default(string), string messageType = default(string), string createdDateTime = default(string), string lastModifiedDateTime = default(string), bool? deleted = default(bool?), string subject = default(string), string summary = default(string), string importance = default(string), string locale = default(string), MessageActionsPayloadFrom from = default(MessageActionsPayloadFrom), MessageActionsPayloadBody body = default(MessageActionsPayloadBody), string attachmentLayout = default(string), IList<MessageActionsPayloadAttachment> attachments = default(IList<MessageActionsPayloadAttachment>), IList<MessageActionsPayloadMention> mentions = default(IList<MessageActionsPayloadMention>), IList<MessageActionsPayloadReaction> reactions = default(IList<MessageActionsPayloadReaction>))
+        /// <param name="linkToMessage">Link to the message on which the action was invoked</param>
+        public MessageActionsPayload(string id = default(string), string replyToId = default(string), string messageType = default(string), string createdDateTime = default(string), string lastModifiedDateTime = default(string), bool? deleted = default(bool?), string subject = default(string), string summary = default(string), string importance = default(string), string locale = default(string), MessageActionsPayloadFrom from = default(MessageActionsPayloadFrom), MessageActionsPayloadBody body = default(MessageActionsPayloadBody), string attachmentLayout = default(string), IList<MessageActionsPayloadAttachment> attachments = default(IList<MessageActionsPayloadAttachment>), IList<MessageActionsPayloadMention> mentions = default(IList<MessageActionsPayloadMention>), IList<MessageActionsPayloadReaction> reactions = default(IList<MessageActionsPayloadReaction>), string linkToMessage = default(string))
```

Comment:
@vman Unfortunately, adding a new (even default) parameter to the end of a method breaks .Net binary compatibility, which is a requirement we have. Even adding an overload won't work here, as all the default argument will create ambiguity. 

In the near term, we need to add an overload that takes a POCO class with all the properties. Maybe we should do that here? something like:
```
public MessageActionPayload(TeamsMessageActionPayloads paylods){ };
public class TeamsMessageActionPayloads
{
  public string Id  { get; set; } = default(string);
   ...
  public string LinkToMessage { get; set; } = default(string); 
}

Reply:
The other (bad) option is to add the property as you've done below, but don't have a constructor parameter for it.